client/extensions/zoninator/components/settings/zone/index.jsx

```diff
@@ -116,7 +139,8 @@ class Zone extends Component {
 					{ translate( 'Edit zone' ) }
 				</HeaderCake>
 
-				{ ! requestingZones && this.renderContent() }
+				{ zone && ! this.locked() && <ZoneLockWarningNotice siteId={ siteId } zoneId={ zoneId } /> }
```

Comment:
As per my comment on the `locked` method, semantically I think this should either be `zone && this.locked() && ...` or `zone && ! this.unlocked() && ...`.